app/routes/admin.settings.tsx

```diff
@@ -357,6 +388,7 @@ export default function Index() {
         }}
       >
         <Settings currencies={loaderData.data.currencies} />
+        <ProductCategoryList categories={loaderData.data.categories} />
```

Comment:
oh, I think you should do this instead

```
<Settings currencies={loaderData.data.currencies} categories={loaderData.data.categories} />
```

The ```<ProductCategoryList  categories={loaderData.data.categories} />``` component should be called within the ```<TabsContent value="product-categories">``` of the Settings.tsx file

There should only be one page component in each controller